### PR TITLE
chore: add a first draft of the config schema.

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

File: lib/config/schema.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        "name": {
+          "type": "string"
+        },
+      },
+      "additionalProperties": false
+    },
+    "apps": {
+      "description": "Properties of the different applications in this project.",
+      "type": "array",
+      "items": {
```
Comment (gkalpak) [inline lib/config/schema.json]: This `items` is missing the `additionalProperties` property. (Not sure if it was intentional, but it seems to be the only one.)

**Comment by jkuri** [comment]
@hansl thanks! LGTM

File: lib/config/schema.json (hunk 1 above)
Comment (hansl) [inline lib/config/schema.json]: It's not intentional for this one, but as it's true by default I'll add it since I don't want it to be extensible.

**Comment by jkuri** [comment]
👍

**Pull request merged**